src/statues/slavers.js

```diff
@@ -1,19 +1,22 @@
 const slaversStatueList = [
   {
+    statueId: 1,
```

Comment:
Can just be `id`.

Reply:
These aren't actually unique as they only help differentiate between different artworks of the same person, but we couldn't think of a better name for them.